### DISCUSSION TITLE
Audio setup in OVOS

### DISCUSSION THREAD

**Discussion by spidermn** [OpenVoiceOS/ovos-buildroot#271] [Q&A]
I am having an issue with audio output after installing OVOS on my setup. My setup is as follow:

rpi 4, 7" rpi touchscreen, Raspiaudio MIC+ V2 (Link to hat - https://raspiaudio.com/product/mic/ )

I followed the instructions on the Raspiaudio site to get the mic to work. To do so I had to install Pulse audio which got the mic working and some sounds to play through the speakers. Navigating through different screens produces clicking sound but when I make a request and OVOS should respond it returns info in the CLI but produces no audio.  Any ideas?  Thanks in advance.

**Comment by suvanbanerjee** [comment]
Ask ovos a question like who is bill gates. then when it should be speaking open pulseaudio manager and check if the volume for ovos audio out stream is muted or not. mostly probably it will be muted so just increase the dial and it will speak 

![image](https://github.com/OpenVoiceOS/ovos-buildroot/assets/104707806/c20999c9-a777-4638-8c42-c9f230116e98)
I don't have the exact screenshot but there will be a python stream when it should speak

Or try the pre-build image for rpi4 
https://ovosimages.ziggyai.online/raspbian/development/

**Comment by wita-sec** [comment]
I also had a couple of issues with OVOS audio (Mic and Speaker) on my RPi4.
Maybe pavucontrol and pulsemixer https://github.com/GeorgeFilipkin/pulsemixer can shed some light on your audio configuration.
For me these tools were very helpful to get audio running.
